app/models/step_by_step_page.rb

```diff
@@ -61,4 +67,28 @@ def auth_bypass_id
   def generate_content_id
     self.content_id ||= SecureRandom.uuid
   end
+
+  def draft_status
+    {
+      name: "draft",
+      text: "Draft",
+      label_class: "label-default"
+    }
+  end
+
+  def live_status
+    {
+      name: "live",
+      text: "Live",
+      label_class: "label-success"
+    }
+  end
+
+  def unpublished_status
+    {
+      name: "unpublished",
```

Comment:
This name could be confusing. We have an "unpublish" button which means something different.